include/robot_interfaces/sensors/pylon_driver.hpp

```diff
@@ -36,17 +36,14 @@ class PylonDriver : public SensorDriver<CameraObservation>
     Pylon::CInstantCamera camera_;
     Pylon::CImageFormatConverter format_converter_;
     Pylon::CPylonImage pylon_image_;
-    Pylon::CTlFactory& tl_factory_;
-    const std::string& device_user_id_to_open_;
 
     /**
-     * @param device_user_id The id of the camera device to open and
+     * @param device_user_id_to_open The id of the camera device to open and
      * grab images from
      */
-    PylonDriver(const std::string& device_user_id)
-        : device_user_id_to_open_(device_user_id),
-          tl_factory_(Pylon::CTlFactory::GetInstance())
+    PylonDriver(const std::string& device_user_id_to_open)
     {
+        Pylon::CTlFactory& tl_factory_ = Pylon::CTlFactory::GetInstance();
```

Comment:
Minor thing, but please remove the trailing `_` to avoid confusion

Reply:
Yep sorry had missed that, fixed now.